### PR TITLE
Add Unicode deprecated characters

### PR DESCRIPTION
**Pull request by xfq**
To sync with https://github.com/w3c/epub-specs/pull/2476/files

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/bp-i18n-specdev/pull/85.html" title="Last updated on Dec 10, 2022, 6:06 AM UTC (b950935)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/bp-i18n-specdev/85/4644204...b950935.html" title="Last updated on Dec 10, 2022, 6:06 AM UTC (b950935)">Diff</a>

### CODE REVIEW

**Review by aphillips — APPROVED** [review]
Looks good to me.

File: index.html [hunk 1]
```diff
@@ -2659,7 +2659,7 @@ <h3>Working with file and path names</h3>
 			<li>The last two code points at the end of the Supplementary Planes (U+1FFFE, U+1FFFF … U+EFFFE, U+EFFFF)</li>
 		</ul></li>
 
-		<li>All Unicode <a href="https://www.unicode.org/Public/UCD/latest/ucd/PropList.txt">deprecated characters</a>.</li>
+		<li>All Unicode <a href="https://www.unicode.org/Public/UCD/latest/ucd/PropList.txt">deprecated characters</a> (search for "Deprecated" in the page).</li>
```
Comment (aphillips) [inline index.html:2662]: Nit: the UCD isn't really a set of "pages", since they are just text files. Perhaps "... in the file" instead?

**Comment by aphillips** [comment]
Thanks for merging this!!

File: index.html (hunk 1 above)
Comment (xfq) [reply index.html:2662]: Thanks. Fixed in https://github.com/w3c/bp-i18n-specdev/commit/1cb452facaa93cccf61401d5827d5cc077f68d68

**Pull request merged**